blog.html

```diff
@@ -0,0 +1,103 @@
+--- 
+title: Mailmeteor Blog - Learn how to send mass emails with Gmail
+description: "We help you get better at Email Marketing with Gmail: detailed Gmail merge tutorials, case studies and opinion pieces."
+
+canonical: https://mailmeteor.com/blog/
+permalink: /blog/
+
+og_title: Mailmeteor Blog - Learn how to send mass emails with Gmail
+og_description: "We help you get better at Email Marketing with Gmail: detailed Gmail merge tutorials, case studies and opinion pieces."
+og_url: https://mailmeteor.com/blog
+og_image: https://mailmeteor.com/assets/img/opengraph/blog-mail-merge-gmail.jpg
+---
+
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: 'en' }}">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
+    <meta name="google-site-verification" content="TShcwwV_esZ98tyJWg6fRMCTDcjUwGnfGBu087vga3M" />
+    <link rel="icon" href="/assets/img/brand/favicon.ico" type="image/x-icon" />
+
+    <title>{{ page.title }}</title>
+    <link rel="canonical" href="{{ page.canonical }}" />
+    <meta name="description" content="{{ page.description }}" />
+
+    <!-- Open graph -->
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="{{ page.og_title }}" />
+    <meta property="og:description" content="{{ page.og_description }}" />
+    <meta
+      name="image"
+      property="og:image"
+      content="{{ page.og_image | default: 'https://mailmeteor.com/assets/img/opengraph/main.jpg' }}"
+    />
+    <meta property="og:url" content="{{ page.og_url }}" />
+    <meta name="twitter:site" content="@mailmeteor" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:url" content="{{ page.og_url }}" />
+    <meta name="twitter:title" content="{{ page.og_title }}" />
+    <meta name="twitter:description" content="{{ page.og_description }}" />
+    <meta
+      name="twitter:image"
+      content="{{ page.og_image | default: 'https://mailmeteor.com/assets/img/opengraph/main.jpg' }}"
+    />
+
+    <!-- Styles -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="/assets/css/main-20210313.css" />
+    <link rel="stylesheet" href="/assets/css/blog.css" />
+  </head>
+  <body class="blogpost">
+    <article itemid="{{ page.canonical }}" itemscope itemtype="https://schema.org/BlogPosting">
```

Comment:
```suggestion
    <div itemid="{{ page.canonical }}" itemscope itemtype="https://schema.org/Blog">
```

J'utiliserais plutôt une `div` à la place de `article`. Côté sémantique, `article` fait totalement sens pour un article de blog, mais pas pour la homepage du blog. Sinon Schema.org, on est à la racine du blog donc c'est `https://schema.org/Blog`